hooks/project-relman/code-review-testing.yml

```diff
@@ -31,6 +31,13 @@ payload:
         else: {}
         then:
           $eval: payload
+      - $if: firedBy == 'triggerHookWithToken'
```

Comment:
Should this be something like `$if: payload`, instead of depending on the mechanism?

Reply:
We should only get these variables when the hook is triggered through a token (as this covers a different workflow than the current one).

I can add extra conditions to make sure we have the variables set, but I think it would remain clearer to have the `firedBy` condition

Reply:
Fine (plus this is following the existing pattern, which I still think is not great but not worth changing now).